### DISCUSSION TITLE
Dynamic Generation of RTR Commands Natively

### DISCUSSION THREAD

**Discussion by modubyk** [CrowdStrike/falconpy#90] [General]
I wanted to open up a thread to see how others have been in addressing this topic in the context of falconpy. 

Is there a way to dynamically generate RTR commands natively? For example "ls" C:\Users and then based on that output for each user profile execute a "get" command for each user profile's C:\Users\<user>\ntuser.dat. 

If the system is online this might be achieved by generating the "get" commands dynamically after enumerating the users from the "ls" command. However, this becomes an issue if the system is **offline** given we won't have the results of the "ls" command. Is there any way to achieve this use-case by dynamic RTR command generation natively?

**Comment by jshcodes** [comment]
In this scenario, do you have access to RTR_Admin?  

Been thinking about this one, and it sounds like an excellent use case for a Custom Script to call with `runscript`.
